starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+h1 a{
+	color: #ffb782;
+	letter-spacing: 2px;
+	font-weight: 500;
+	text-decoration: none;
+	font-size: 48px;
+	
+}
+
+body {
+	font-family: Lato;
+	padding: 30px 925px 0px 50px;
+}
+
+nav a{
+	font-size: 22px;
+	color: #c4c4c4;
+	padding: 0px 5px 0px 5px;

```

Comment:
shorthand way to write this: `padding: 0 5px;`. That sets the top and bottom to 0, left and right to 5px.